ipaserver/plugins/baseuser.py

```diff
@@ -526,6 +567,82 @@ def convert_attribute_members(self, entry_attrs, *keys, **options):
         except KeyError:
             pass
 
+    def handle_subordinate_ids(self, ldap, dn, entry_attrs):
+        """Handle ipaSubordinateId object class
+        """
+        obj_classes = entry_attrs.get("objectclass")
+        new_subuid = entry_attrs.single_value.get("ipasubuidnumber")
+        new_subgid = entry_attrs.single_value.get("ipasubgidnumber")
+
+        # entry has object class ipaSubordinateId
+        # default to auto-assigment of subuids
+        if obj_classes is not None and "ipasubordinateid" in obj_classes:
```

Comment:
There is a helper for this, self.obj.has_objectclass()

Reply:
Thanks, that's an excellent suggestion.